packages/fxa-auth-server/scripts/create-mock-iap-subscriptions.ts

```diff
@@ -0,0 +1,69 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+import program from 'commander';
+
+import { setupProcessingTaskObjects } from '../lib/payments/processing-tasks-setup';
+
+const pckg = require('../package.json');
+import * as mockIapSubscriptions from './create-mock-iap-subscriptions/mock-iap-subscriptions.json';
+import { MockIapSubscriptionManager } from './create-mock-iap-subscriptions/mock-iap-subscription-manager';
+
+export async function init() {
+  program
+    .version(pckg.version)
+    .option('-c, --config [config]', 'Configuration to use. Ex. dev')
+    .option('-u, --uid [uid]', 'User ID to create subscription for')
+    .option(
+      '--appStoreProductId [appStoreProductId]',
+      'Any string. Should be present in a Stripe plan\'s "appStoreProductIds" metadata field'
```

Comment:
Nit: Might want to omit references to Stripe metadata, since we're moving to Firestore to store this config, and these identifiers ultimately come from Google Play or the App Store.

Reply:
Removed the word "Stripe"